template-xml/domain/src/main/java/co/nimblehq/template/xml/domain/repository/Repository.kt

```diff
@@ -1,6 +1,6 @@
-package co.nimblehq.template.domain.repository
+package co.nimblehq.template.xml.domain.repository
```

Comment:
I don't think we need to move the `data` and `domain` module to the new package `co.nimblehq.template.xml`. I think  `co.nimblehq.template` is enough. What do you think @Wadeewee @Tuubz?

Reply:
If the package for `data` & `domain` are not aligned with `app`, then we will have to make changes to the script, no? 💭

Reply:
> I don't think we need to move the data and domain module to the new package co.nimblehq.template.xml

@luongvo Shouldn't we name the package to make it consistent with the `module's name`

Reply:
@Wadeewee @Tuubz Alright, makes sense cuz we will have a different package for compose template too 🤔